rubrik_cdm/cluster.py

```diff
@@ -606,3 +608,58 @@ def read_only_authorization(self, username, timeout=15):
 
         self.log("read_only_authorization: Granting read-only privilages to user '{}'.".format(username))
         return self.post("internal", "/authorization/role/read_only_admin", config, timeout)
+
+    def cluster_node_id(self):
+        """ Returns a list of node ids from all the nodes in the cluster.
+
+                Arguments:
+                    None
+
+                Keyword Arguments:
+                    None
+
+                Returns:
+                    dict -- The full API response from `POST /internal/authorization/role/read_only_admin`.
+                """
+        self.log("cluster_node_id - Getting all the node ids from the from the cluster")
+        node_id = self.get('internal', '/node')
+        return node_id
```

Comment:
To simplify the code, you can just skip assigning the API call to a variable and just `return self.get("internal", "/node")`